core/views.py

```diff
@@ -15,13 +15,13 @@ def __init__(self):
         super(HashedNameServiceClient, self).__init__()
 
     def name_to_hash(self, name):
-        return requests.get(f"http://35.178.109.215:5000/name2hash/{urllib.parse.quote(name)}").text
+        return requests.get(f"https://cbp-id-provider.herokuapp.com/name2hash/{urllib.parse.quote(name)}").text
 
     def hash_to_name(arg, hash):
-        return urllib.parse.unquote(requests.get(f"http://35.178.109.215:5000/hash2name/{hash}").text)
+        return urllib.parse.unquote(requests.get(f"https://cbp-id-provider.herokuapp.com/hash2name/{hash}").text)
```

Comment:
@gusmith I tried using the hash I received from `name2hash` here but I got a 404.
https://cbp-id-provider.herokuapp.com/hash2name/3AW8RMXKeG9NK4HQRopU

Reply:
@RyanBalfanz The hash 3AW8RMXKeG9NK4HQRopU is from JOHNIE BAUMKIRCHNER who is not in the bank B, so his hash is unknown to bank B.

As a reminder, we ask the hash of a user in bank A (JOHNIE BAUMKIRCHNER returning 3AW8RMXKeG9NK4HQRopU), then we do the query to the engine which returns a umber of hashes and score of user in bank B. We finally request more information about these users in bank B. Because we were expecting them to be in bank B for the demo, only hashes of user in bank B can be used in the endpoints `hash2name` and `hash2info`.

You can try with the hashes 
 - [adzhPj8Ck3xCnrguGhdO](https://cbp-id-provider.herokuapp.com/hash2name/adzhPj8Ck3xCnrguGhdO)
 - [cX6Vm8qcsqpDQJtzFcU7](https://cbp-id-provider.herokuapp.com/hash2name/cX6Vm8qcsqpDQJtzFcU7)
 - [iRKG4uYxKRWuBa6psK22](https://cbp-id-provider.herokuapp.com/hash2name/iRKG4uYxKRWuBa6psK22)
 - [KeS3H4FFWfB4bdZnliev](https://cbp-id-provider.herokuapp.com/hash2name/KeS3H4FFWfB4bdZnliev)
Which are the four results from our search.